vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ReferenceContext.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+import edu.asu.diging.vspace.core.model.IReference;
+
+public class ReferenceContext {
+    
+    private ICitationStyle citationStyle;
+    
+    private IReference reference;
+
+    public ReferenceContext(ICitationStyle citationStyle, IReference reference) {
+        this.citationStyle = citationStyle;
+        this.reference = reference;
+    }
+    
+    public IReferenceMetadata getReferenceMetadataConverter() {
+        //We can introduce switch case for different types of metadata/citation style to return appropriate IBiblioMetadata
+        //For now its default
```

Comment:
no, no switch cases. I think I commented on that before. The different citation styles should be in their own classes that all implment a comment interface and then get loaded into this class (in a map). Then they can be used via the IDs they register with.